### ISSUE TITLE
Bug: Kit pageviews miscounting

### ISSUE THREAD

**Issue by Mimisss** [FortAwesome/Font-Awesome#19941]
### Bug description

It is now clear to us that the pageviews of our (free) kit are miscounted.

First, out of the blue, we received warnings about using more than the allowed monthly pageviews. That was quite weird because (a) we have never had reached the 10000 limit before and (b) our web app is subscription based and we monitor our users' activity closely. Also, our kit is restricted for use in our two domains only.

We decided to give fa's counting algorithm the benefit of the doubt so we created a new kit and started monitoring pageview count. While it worked for a few months (with total monthly pageviews well below 10000) then suddenly we were warned once again.

This time we decided to stop using the kit and self-host fa, which we implemented. We took a note of 14988 total pageviews for the range between 2023-10-01 and 2023-10-08. 

**Surprisingly, today, 2023-10-10, the  total pageviews are 14991! Obviously, your algorithm keeps counting even though the kit is not used any more!**

So, please, check your counting algorithm again. We sincerely hope this is not just a trick to have users upgrade to paid kits. 

### Font Awesome version

v.5.x

### Application and Operating System

- Windows Server 2012 R2
- IIS Web Server
- ASP.NET Core 3.1 Web Application

### Web bug report checklist

- [X] I have included a test case because my odds go _way_ up that the team can fix this when I do
- [X] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate

**Comment by pixeline** [comment]
Same here - FA reports a huge number of pageviews that contradicts what our Google Analytics pageviews report (~ tenfold diference). I suggest you at least disclose how you count pageviews.

**Comment by robmadole** [comment]
@pixeline good point about disclosure. I don't think we've done this. I'm adding a task to put this somewhere.

Regarding Google Analytics, we simply don't count like that tool does. A pageview for Kits is any request that returns a 200 or 304 status code. That's it, nothing else.

We don't filter bot traffic, you pay for development requests (to localhost, etc.); it really is per-request tracking.

**Comment by Mimisss** [comment]
Thanks for your answer but honestly you should rethink about **why** you count and **how** you count. And probably drop the count-based billing approach altogether.

**Comment by robmadole** [comment]
> And probably drop the count-based billing approach altogether.

Interesting, what do you propose @Mimisss ?

**Comment by Mimisss** [comment]
> > And probably drop the count-based billing approach altogether.
> 
> Interesting, what do you propose [@Mimisss](https://github.com/Mimisss) ?

Since you are unwilling to verify that your page view counting works correctly may I suggest you switch to fixed-price kits. How you charge for them is up to you (e.g. monthly, annually, per icon style e.t.c.). That would be more viable especially for non-SPA web applications that dynamically serve web pages on every web request.

**Comment by robmadole** [comment]
Thanks for the feedback @Mimisss, appreciate it.
